exercises/APE1/0106.tex

```diff
@@ -0,0 +1,40 @@
+In order to answer this question, we are going to decompose our reasoning into three steps.
+First of all, we will consider that both players have a pure strategy. Then, that one player has a pure strategy and the other one has a randomized strategy. Finally, we will assume that both players have a randomized strategy.
+
+\subsection*{Pure strategy vs. pure strategy}
+
+We can directly see that $(x_1, y_2)$ is not an equilibrium. Indeed, both players can increase their payoff from deviating alone from their strategy. Player 2 will gain 2 instead of 1 if he deviates alone to $x_2$ and player 1 will gain 5 instead of 4 if he deviates alone to $y_1$.
+
+Since the game is symmetric, $(y_1, x_2)$ is neither an equilibrium.
```

Comment:
"is neither an equilibrium" -> "is not an equilibrium either"
Not sure that is is wrong but it really sounds weird with "neither"